src/plugins/data_source/server/legacy/configure_legacy_client.ts

```diff
@@ -3,27 +3,38 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Client } from 'elasticsearch';
+import { Client } from '@opensearch-project/opensearch-next';
```

Comment:
This dependency update is not needed. We are using `elasticsearch` everywhere in `data_source/server/legacy`, that's to support MD in legacy client use case

Reply:
We may want a follow-up PR for `main` then - that's in the original PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3058/files#diff-442cda9a371f9aa2ce417631ddf7a8b80f9ec6ade15676b1cfe390f1c25b3fe7R6

Reply:
Oh my bad, I didn't see the new lines below. No need to change.  `elasticsearch` is being added in line 7.